homeassistant/components/ihc/config_flow.py

```diff
@@ -0,0 +1,86 @@
+"""Config flow for IHC integration."""
+import logging
+
+from ihcsdk.ihccontroller import IHCController
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.const import CONF_PASSWORD, CONF_URL, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import AbortFlow, FlowResult
+
+from .const import CONF_AUTOSETUP, DOMAIN
+from .util import get_controller_serial
+
+CONFIG_FLOW_VERSION = 1
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_URL, default="http://192.168.1.3"): str,
```

Comment:
Can there really be a default IP?

Reply:
The default IP is the IP an ihc controller will have in a default setup. It is also used as default by LK (the ihc controller manufacture) in the software for the controller. I know that may look strange but it will have this IP event if you don't have the 192.168.1.x network. You can change it in the controller setup, but you need to do it from this network originally (or use a usb connection), so I think most users will leave at this IP